falcon/errors.py

```diff
@@ -169,6 +177,10 @@ class WebSocketServerError(WebSocketDisconnected):
     pass
 
 
+Kwargs = Union[str, int, None]
```

Comment:
I'd rather leave it where it is, because that's the only module where Kwargs are actually used, so in my reasoning i thought that we should keep the code as close to the usage as possible. Renamed.